File: Service_eHospital_DongNai_A_NSTL/Implement/NS_DM_PHUCAPTRUCService.cs
namespace Service_eHospital_DongNai_A_NSTL.Implement
{
    public class NS_DM_PHUCAPTRUCService : BaseService<NS_DM_PHUCAPTRUC, INS_DM_PHUCAPTRUCRepository>
    , INS_DM_PHUCAPTRUCService
    {
    private readonly INS_DM_PHUCAPTRUCRepository _NS_DM_PHUCAPTRUCRepository;
    public NS_DM_PHUCAPTRUCService(INS_DM_PHUCAPTRUCRepository NS_DM_PHUCAPTRUCRepository) : base(NS_DM_PHUCAPTRUCRepository)
    {
    _NS_DM_PHUCAPTRUCRepository = NS_DM_PHUCAPTRUCRepository;
    }
    }
    }

File: CRM/src/app/shared/eHospital_DongNai_A_NSTL/NS_DM_PHUCAPTRUC.service.ts
import { Injectable } from '@angular/core';
import { HttpClient, HttpParams } from '@angular/common/http';
import { environment } from 'src/environments/environment';
import { NS_DM_PHUCAPTRUC } from './NS_DM_PHUCAPTRUC.model';
import { BaseService } from './Base.service';
@Injectable({
    providedIn: 'root'
})
export class NS_DM_PHUCAPTRUCService extends BaseService{
    DisplayColumns001: string[] = ['Save', 'STT', 'ID', 'MaKhoa', 'TiLeThuong'];

    List: NS_DM_PHUCAPTRUC[] | undefined;
    ListFilter: NS_DM_PHUCAPTRUC[] | undefined;
    FormData!: NS_DM_PHUCAPTRUC;
    constructor(public httpClient: HttpClient) {
        super(httpClient);
        this.Controller = "NS_DM_PHUCAPTRUC";
    }
}

